### ISSUE TITLE
Problem with LED_TACTILE_BUTTONSFE

### ISSUE THREAD

**Issue by FubarGR** [EagleWorks/eagle#1]
LED_TACTILE_BUTTONSFE has the led anode and cathode swapped!

**Comment by EagleWorks** [comment]
Hi, thanks for the report!  I haven't used this part myself (I had already made the other package and added the one based on the break-out at a later time) but upon review it does seem that while the pins were correct, the silkscreen was incorrect for that package (probably messed up a rotate somewhere along the line).  If you've already made boards with this design, rotate the button 180 degrees for proper operation.  Let me know if you got bit by this in a project.  Library file updated to fix this. Edit: well, will be shortly. Edit 2: done.

**Comment by FubarGR** [comment]
Hi!

Sorry for the late reply and thank you, your part saved me quite some time!

When I found about the problem, I had already routed the board so I  
simply rotated the diode symbol on the silkscreen.

There was also another issue, the plastic feet on the bottom of the  
button didn't align with the holes. But this could be an issue with my  
buttons, they are Ninigi branded, not from Sparkfun. I simply snipped  
the plastic nubs off with sidecutters and the problem was solved!

The board is here on OSHpark: https://oshpark.com/shared_projects/vzgGvprD

And this is how it looks in real life:
http://tinypic.com/r/mjbvqg/9
http://tinypic.com/r/2vlmzxk/9
http://tinypic.com/r/ztsoi/9

Quoting EagleWorks notifications@github.com:
